src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/database/daos/ContentDao.kt

```diff
@@ -20,6 +20,17 @@ class ContentDao(
                 }
     }
 
+    fun fetchVerseByCollectionIdAndStart(collectionId: Int, start: Int, dsl: DSLContext = instanceDsl): ContentEntity? {
+        return dsl
+                .select()
+                .from(CONTENT_ENTITY)
+                .where(CONTENT_ENTITY.COLLECTION_FK.eq(collectionId))
+                .and(CONTENT_ENTITY.START.eq(start))
+                .and(CONTENT_ENTITY.LABEL.eq("verse")) // TODO: Probably shouldn't be hardcoded
```

Comment:
Re todo: I'd prefer to have a `fetchContentByRcAndCollectionAndStart` filtered on RC_FK instead of label.